code/modules/antagonists/abductor/abductor.dm

```diff
@@ -148,7 +148,7 @@
 	var/mob/living/carbon/human/H = owner.current
 	var/gear = tgui_alert(admin,"Agent or Scientist Gear", "Gear", list("Agent", "Scientist"))
 	if(gear)
-		if(gear=="Agent")
+		if(gear =="Agent")
```

Comment:
Looks like this one is still broken

Reply:
hate to keep tacking on more exceptions. ill look for a cleaner way to do it

Reply:
There's no clean way to do it as long as we are using greps and not a sane AST based system, don't bother

Reply:
added a few lines to catch this case (and where it's not a string ie "==")